### ISSUE TITLE
Default email subject for leads, customers, etc.

### ISSUE THREAD

**Issue by gbard** [frappe/erpnext#10092]
(Feature request)

Hello,

There should be a way to either set a default e-mail subject when sending e-mails to leads, customers, etc. in the e-mail templates or let the system choose the default e-mail subject ie: "Lead: LEAD-00675" if no default subject was defined for the choosen template. As of now, we need to manually change the subject for every leads.

Thanks!

G.

**Comment by AleCVERP** [comment]
Yes please, i have the same problem with subject and message. I have to change manually.
Thanks

**Comment by Ruttie** [comment]
Me too. Would be great if this would be possible.
Thank you.

**Comment by jhquiroga** [comment]
Same here.. this would be a great feature, I've been trying to accomplish this using Standard reply but it doesn't accept jinja tags in Subject, besides Standard reply sends email from default account, not the user who send it, and the reply-to goes to that email

**Comment by joolsr** [comment]
I have a feeling that I read that jinja tags will or now are acceptable in the Subject line of an email, though I can't find the commit that mentions this.

**Comment by sbkolate** [comment]
@gbard 
There are some other feature request with standard reply. 
Please check https://github.com/frappe/erpnext/issues/7921
Let me know if this solve your problem, as it is long overdue, we can collaborate and fix it.
